### ISSUE TITLE
Update ANDI favelet link on dhs.gov/508-tools

### ISSUE THREAD

**Issue by drewnielson** [Section508Coordinators/DHSA-TT-200-A#33]
Change from: 
`javascript:void((function()%7bandiScript=document.createElement('script');andiScript.setAttribute('src','https://www.dhs.gov/xlibrary/oast/ANDI/andi.js');document.body.appendChild(andiScript)%7d)());`
Change to: 
`javascript:void((function(){andiScript=document.createElement('script');andiScript.setAttribute('src','https://www.dhs.gov/xlibrary/oast/ANDI/andi.js');document.body.appendChild(andiScript)})());`

In other words, change `%7b` to `{` and change `%7d` to `}`

**Comment by drewnielson** [comment]
Fixed
